keyboards/k_type/led.c

```diff
@@ -0,0 +1,57 @@
+/* Copyright 2021 Andrew Fahmy
```

Comment:
This file should probably be renamed to avoid possible confusion with the core led.c, which deals with indicator (lock) LEDs, although it is probably not used for the K-Type.